TestCases/optimization_euler/equivalentarea_naca64206/NACA64206_adjoint.cfg

```
@@ -0,0 +1,159 @@
+
+% ------------- DIRECT, ADJOINT, AND LINEARIZED PROBLEM DEFINITION ------------%
+%
+% Physical governing equations (EULER, NAVIER_STOKES)
+SOLVER= EULER
+%
+% Mathematical problem (DIRECT, CONTINUOUS_ADJOINT, DISCRETE_ADJOINT)
+MATH_PROBLEM= DISCRETE_ADJOINT
+%
+% Read binary restart files (YES, NO)
+READ_BINARY_RESTART= NO
+
+% -------------------- COMPRESSIBLE FREE-STREAM DEFINITION --------------------%
+%
+% Mach number (non-dimensional, based on the free-stream values)
+MACH_NUMBER= 1.6
+%
+% Angle of attack (degrees)
+AOA= 0.0
+%
+% Side-slip angle (degrees)
+SIDESLIP_ANGLE= 0.0
+%
+% Free-stream pressure (101325.0 N/m^2 by default, only for Euler equations)
+FREESTREAM_PRESSURE= 11662.0
+%
+% Free-stream temperature (288.15 K by default)
+FREESTREAM_TEMPERATURE= 216.6499
+
+REF_ORIGIN_MOMENT_X = 0.25
+REF_ORIGIN_MOMENT_Y = 0.00
+REF_ORIGIN_MOMENT_Z = 0.00
+REF_LENGTH= 1
+REF_AREA= 0
+REF_DIMENSIONALIZATION= FREESTREAM_PRESS_EQ_ONE
+
+% ------------------------ SUPERSONIC SIMULATION ------------------------------%
+% MARKER_NEARFIELD needs to be defined on a circumferential boundary within
+% calculation domain so that it captures pressure disturbance from the model.
+% The boundary should have a structured grid with the same number of nodes
+% along each azimuthal angle.
+% To run inverse design using target equivalent area, TargetEA.dat is required.
+%
+% Evaluate equivalent area on the Near-Field (NO, YES)
+% 
+EQUIV_AREA= YES
+%
+% Integration limits of the equivalent area ( xmin, xmax, Dist_NearField )
+EA_INT_LIMIT= ( 234.854, 313.354, 220.38)
+%
+% Equivalent area scale factor ( EA should be ~ force based objective functions )
+EA_SCALE_FACTOR= 1.0
+%
+% Fix an azimuthal line due to misalignments of the near-field
+FIX_AZIMUTHAL_LINE= 90.0
+%
+% Drag weight in sonic boom Objective Function (from 0.0 to 1.0)
+DRAG_IN_SONICBOOM= 0.0
+
+% ----------------------- BOUNDARY CONDITION DEFINITION -----------------------%
+%
+MARKER_EULER= ( Wing )
+MARKER_FAR= ( Side )
+MARKER_SUPERSONIC_INLET=(Inlet,216.6499,11662.0,472.1153,0,0)
+MARKER_SUPERSONIC_OUTLET=(Outlet)
+MARKER_SYM= ( Symmetry )
+MARKER_PLOTTING= ( Wing )
+MARKER_MONITORING= ( Wing )
+%
+% Near-Field boundary marker(s) (NONE = no marker)
+MARKER_NEARFIELD= ( Nearfield )
+
+% ------------- COMMON PARAMETERS TO DEFINE THE NUMERICAL METHOD --------------%
+%
+NUM_METHOD_GRAD= GREEN_GAUSS
+
+% Objective function in gradient evaluation  (DRAG, LIFT, SIDEFORCE, MOMENT_X,
+%                                             MOMENT_Y, MOMENT_Z, EFFICIENCY,
+%                                             EQUIVALENT_AREA, NEARFIELD_PRESSURE,
+%                                             FORCE_X, FORCE_Y, FORCE_Z, THRUST,
+%                                             TORQUE, FREE_SURFACE, TOTAL_HEATFLUX,
+%                                             MAXIMUM_HEATFLUX, INVERSE_DESIGN_PRESSURE,
+%                                             INVERSE_DESIGN_HEATFLUX)
```

```suggestion
% Objective function in gradient evaluation
```
We are trying to avoid listing options in the examples because it makes it difficult to keep everything up to date.
We use the config_template to list the options, and the examples to highlight / test particular ones.
At the moment if you run SU2 and ask for a "wrong" option it will print all the valid possibilities.